back/src/main/resources/properties/jwt.yml

```
@@ -0,0 +1,14 @@
+  config.activate.on-profile: local
+jwt:
+  secret: ${JWT_SECRET}
+  expiration: ${JWT_ACCESS_EXPIRATION}
+
+---
+# test 환경
+spring:
+  config.activate.on-profile: test
+jwt:
+  secret: ${JWT_SECRET}
+  expiration: ${JWT_ACCESS_EXPIRATION}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)